static/app/components/assistant/getGuidesContent.tsx

```diff
@@ -274,6 +274,28 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
         },
       ],
     },
+    {
+      guide: 'new_page_filters',
+      requiredTargets: ['new_page_filter_button'],
+      expectedTargets: ['new_page_filter_pin'],
+      steps: [
+        {
+          title: t('Selection Filters have moved!'),
+          target: 'new_page_filter_button',
+          description: t(
+            'Selection filters have teleported from the top of the page to live closer to the content they are filtering on!'
+          ),
+          nextText: t('Tell me more'),
+        },
+        {
+          title: t('Explicit memory'),
+          target: 'new_page_filter_pin',
+          description: t(
+            'Selection Filters no longer will implicitly be remembered between sessions. You can now "Pin" a filter to remember it'
```

Comment:
maybe just personal feeling here, but it makes more sense in my head to write it as `Selection filters will no longer be implicitly...`